<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastSwitchDlg</class>
 <widget class="QDialog" name="ContrastSwitchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>719</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxGrayMask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>转换偏移</string>
   </property>
   <widget class="QSlider" name="horizontalSliderGrayMask">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>61</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGrayMask">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>41</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddGrayMask">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>61</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubGrayMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>719</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxGainMask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>转换强度</string>
   </property>
   <widget class="QSlider" name="horizontalSliderGainMask">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>61</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGainMask">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>41</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddGainMask">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>61</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubGainMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 108, 108);</string>
   </property>
   <property name="text">
    <string>对比度转换</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
